fendermint/docker/builder.ci.Dockerfile

```diff
@@ -70,7 +70,8 @@ RUN set -eux; \
   *) echo >&2 "unsupported architecture: ${TARGETARCH}"; exit 1 ;; \
   esac; \
   rustup show ; \
-  cargo build --release -p fendermint_app --target ${ARCH}-unknown-linux-gnu
+  cargo build --release -p fendermint_app --target ${ARCH}-unknown-linux-gnu &&\
+  cargo build --release -p ipc-cli --target ${ARCH}-unknown-linux-gnu
```

Comment:
Let's add `--locked` here too. I added in one of my draft PRs. I think if the `cargo install` below has it, best if this one does too.

Reply:
Added. Thanks!